shared/src/business/utilities/htmlGenerator/documents/order.scss

```diff
@@ -16,10 +16,18 @@
       line-height: 20px;
     }
 
-    ol ol {
+    ol ol ol ol, ol ol ol ol ol ol ol {
+      list-style-type: decimal;
+    }
+
+    ol ol, ol ol ol ol ol, ol ol ol ol ol ol ol ol {
       list-style-type: lower-alpha;
     }
 
+    ol ol ol, ol ol ol ol ol ol, ol ol ol ol ol ol ol ol ol {
+      list-style-type: lower-roman;
+    }
+
```

Comment:
Can't get a counter to work with the pdf generator, lmk if anyone knows of a better way to do this